Contest_API.md

```diff
@@ -1448,23 +1488,22 @@ For some common award cases the following IDs should be used.
 | group-winner-\<id>        | Current leader(s) in group \<id>. Empty if no team has scored.                                                             | Winner(s) of group \<id>.                 |
 | organization-winner-\<id> | Current leader(s) of organization \<id>. Empty if no team has scored.                                                      | Winner(s) of organization \<id>.          | Not useful in contest with only one team per organization (e.g. the WF).
 
-#### POST, PATCH, and DELETE awards
+#### POST, PUT, PATCH, and DELETE awards
 
-Clients with the `admin` role may make changes to awards using the normal [HTTP methods](#http-methods) as specified above. Specifically, they can POST new awards, PATCH one or more attributes, or DELETE an existing award. All requests must include the id attribute.
+Clients with the `admin` role may make changes to awards using the normal [HTTP methods](#http-methods) as specified above. Specifically, they can POST new awards, create or replace one with a known id via PUT, PATCH one or more attributes, or DELETE an existing award.
 
 The server may be configured to manage (assign or update) some award ids, and may block clients from modifying them. However, if a client is able to modify an award it must assume that it is responsible for managing that award id unless and until it sees an indication that something else is now managing that award - either a change that it did not request, or a future modification fails.
 
 For example, the server may be configured to assign the `winner` award and not allow any client to modify it. The same server may assign `*-medal` awards by default, but allow clients to modify them. Once a client modifies any of the `*-medal` awards, it is responsible for updating it if anything changes. Likewise, the client could add any arbitrary awards like `first-submission-for-country-*` and would be responsible for managing these.
 
-The request must fail with a 4xx HTTP status code if any of the following happens:
+The request must fail with a 4xx error code if any of the following happens:
 
-* The request doesn't include an award id.
-* A POST when the award id already exists.
-* A POST that is missing one of the required attributes (`citation` and `team_ids`).
-* A PATCH on an award id that doesn't exist.
+* A POST that includes an id.
+* A PUT, PATCH, or DELETE on an award that doesn't exist.
```

Comment:
PUT on an award that doesn't exist *is* allowed.

Reply:
Fixed.